util/query.py

```diff
@@ -336,12 +342,47 @@ def _filterOutPurgatory(nftvols: dict, chainID: int) -> dict:
       filtered_nftvols: list of [basetoken_addr][nft_addr]:vol_amt
     """
     bad_dids = _didsInPurgatory()
-    filtered_pools = {}
+    filtered_nfts: Dict[str, Dict[str, float]] = {}
     for basetoken_addr in nftvols:
         for nft_addr in nftvols[basetoken_addr]:
             if oceanutil.calcDID(nft_addr, chainID) not in bad_dids:
-                filtered_pools[basetoken_addr] = nftvols[basetoken_addr]
-    return filtered_pools
+                filtered_nfts[basetoken_addr] = nftvols[basetoken_addr]
+    return filtered_nfts
+
+
+@enforce_types
+def _filterOutNonMarketAssets(nftvols: dict, chainID: int) -> dict:
+    """
+    @description
+      Return nfts that belong to the Ocean marketplace
+
+    @arguments
+      nftvols: dict of [basetoken_addr][nft_addr]:vol_amt
+
+    @return
+      filtered_nftvols: list of [basetoken_addr][nft_addr]:vol_amt
+    """
+    filtered_nfts: Dict[str, Dict[str, float]] = {}
+    didList = []
```

Comment:
Problems: "didList" label is too vague, and camelCase. 

Fix by: rename to nft_dids. This parallels "nftvols".

Finally, it should be a set not a list, to make clear that there might be duplicates.